tests/test_util.py

```diff
@@ -1,7 +1,28 @@
+import os.path
+import subprocess
+
 import attr
 import pytest
 
 from labgrid.util import diff_dict, flat_dict, filter_dict
+from labgrid.util.ssh import ForwardError, SSHConnection, sshmanager
+from labgrid.util.proxy import proxymanager
+from labgrid.util.managedfile import ManagedFile
+from labgrid.driver.exception import ExecutionError
+from labgrid.resource.serialport import NetworkSerialPort
+from labgrid.resource.common import Resource, NetworkResource
+
+@pytest.fixture
+def connection_localhost():
+    con = SSHConnection("localhost")
+    con.connect()
+    yield con
+    con.disconnect()
+
+@pytest.fixture
+def sshmanager_fix():
+    yield sshmanager
+    sshmanager.close_all()
```

Comment:
Is this still needed after "pytestplugin/fixtures: close all ssh connections after the test ends"?

Reply:
Yes, closing the SSH Connections as implemented in the mentioned commit only works with the `env` fixture. Since the tests for labgrid itself don't use this fixture, we have to do this explicitly ourselves.